File: DB e BACKEND/API_EsercitazioneV2/API_Esercitazione/Models/Studente.cs
namespace API_Esercitazione.Models
{
    public class Studente
    {
        public string cod_fiscale { get; set; }
        public string nome { get; set; }
        public string cognome { get; set; }
        public DateTime data_nascita { get; set; }
        public string comune_nascita { get; set; }
        public string num_tel { get; set; }
        public string indirizzo_res { get; set; }
        public string civico_res { get; set; }
        public int cap_res { get; set; }
        public DateTime doi { get; set; }
        public DateTime dou { get; set; }

    }
}

File: src/models/studente.ts
export interface Studente{
    cod_fiscale: string,
    nome: string,
    cognome: string,
    data_nascita: Date,
    comune_nascita: string,
    num_tel: string,
    indirizzo_res:string,
    civico_res:string,
    cap_res: number,
    doi?:Date,
    dou?:Date 
}